CSS/wishlist.css

```diff
@@ -0,0 +1,96 @@
+@import url(/CSS/style.css);
+
+.wishlist_container,
+.btn_container,
+.price_container,
+.wishlist_card_details,
+.wishlist_icon {
+    display: flex;
+}
+
+.wishlist_card_details,
+.wishlist_icon,
+.btn_container {
+    justify-content: center;
+}
+
+.e_secondary_btn,
+.wishlist_card_details,
+.price_tag {
+    font-size: 1.5rem;
+}
+
+.title,
+.price_tag,
+.e_secondary_btn {
+    font-weight: bold;
+}
+
+.wishlist_card,
+.headline {
+    text-align: center;
+}
+
+.price_tag,
+.e_secondary_btn,
+.wishlist_card_details {
+    padding: 0.5rem;
+}
+
+.wishlist_container {
+    background-color: #f9fafb;
+    padding: 2rem;
+    justify-content: space-around;
+    flex-wrap: wrap;
+}
+
+.headline {
+    font-size: 2rem;
+    font-family: var(--secondary-font);
+    margin-top: 1rem;
+}
+
+.wishlist_card {
+    width: 20rem;
+    box-shadow: 1px 1px 4px 1px var(--secondary-color);
+    border-radius: 0.5rem;
+    background-color: white;
+}
+
+.wishlist_img {
+    border-bottom: 2px solid var(--boder-color);
+    position: relative;
+}
+
+.wishlist_icon {
+    position: absolute;
+    right: 0.5rem;
+    top: 1rem;
+    align-items: center;
+    height: 2rem;
+    width: 2rem;
+    border: 1px solid var(--boder-color);
+    border-radius: 50%;
+}
+
+.price_container {
+    flex-direction: column;
+}
+
+.e_secondary_btn {
+    width: 100%;
+    border-radius: 0.4rem;
+    color: #60a5fa;
+    background-color: var(--white-color);
+    border: 1px solid #60a5fa;
+    background-color: var(--white-color);
+}
+
+.e_secondary_btn:hover {
+    background-color: #60a5fa;
+    color: white;
```

Comment:
Do properly align the file before merging. You can use the prettier or beautify extension for that

Reply:
okay will do that